modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/shared/components/toolbar/UpperToolbar.js

```diff
@@ -65,6 +65,13 @@ export default function UpperToolbar({displayNames, isView, languageIds}) {
 		translations,
 		version,
 	} = useContext(DefinitionBuilderContext);
+
+	function _setAlert(alertMessage, alertType, showAlert) {
+		setAlertMessage(alertMessage);
```

Comment:
Hi @M-IgorFranca I believe that you need to check if the `alertMessage` exists before execute `setAlertMessage(alertMessage);`. Could yout check if it's necessary?